docs/integrations/common-workflow-steps/build-cache/cache-pull.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cache Pull 
+description: Enhance your development process with Cache Pull to quickly retrieve and reuse stored data, boosting efficiency and performance.
+tags: [cache pull, efficiency, dependencies, cache structure]
+sidebar_position: 2
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Cache Pull
+
+[**Cache Push**](./cache-push) uploads the cache archive file to a remote location, as we explained in detail in the **Cache Push** step. On the other hand, **Cache Pull** downloads and extracts that archive file in the build pipeline, restoring all files and folders to their original locations.
```

Comment:
Links should be the exact path.

Wrong:
`[**Cache Push**](./cache-push)`

Correct:
`[**Cache Push**](/workflows/common-workflow-steps/cache-push)`